client/state/order-transactions/constants.js

```diff
@@ -1,8 +1,14 @@
+export const SUCCESS = 'ORDER_TRANSACTION_STATUS_SUCCESS';
```

Comment:
Exporting and importing these directly preserves their string literal types for TypeScript. For example, `SUCCESS` now has the type `'ORDER_TRANSACTION_STATUS_SUCCESS'` but `ORDER_TRANSACTION_STATUS.SUCCESS` has the type `string`.